src/origin.rs

```diff
@@ -60,18 +63,27 @@ pub struct OriginState {
 
 #[cfg(feature = "2d")]
 #[cfg(not(feature = "3d"))]
-type WithMesh = With<Mesh2dHandle>;
+type WithSvg = With<Svg2d>;
+#[cfg(not(feature = "2d"))]
+#[cfg(feature = "3d")]
+type WithSvg = With<Svg3d>;
+#[cfg(all(feature = "2d", feature = "3d"))]
+type WithSvg = Or<(With<Svg2d>, With<Svg3d>)>;
+
+#[cfg(feature = "2d")]
+#[cfg(not(feature = "3d"))]
+type WithMesh = With<Mesh2d>;
 #[cfg(not(feature = "2d"))]
 #[cfg(feature = "3d")]
-type WithMesh = With<Handle<Mesh>>;
+type WithMesh = With<Mesh3d>;
 #[cfg(all(feature = "2d", feature = "3d"))]
-type WithMesh = Or<(With<Mesh2dHandle>, With<Handle<Mesh>>)>;
+type WithMesh = Or<(With<Mesh2d>, With<Mesh3d>)>;
 
 /// Checkes if a "new" SVG bundle was added by looking for a missing `OriginState`
 /// and then adds it to the entity.
 pub fn add_origin_state(
```

Comment:
I think, we need this only for backwards compatibility with `Bundle`s and not anymore for the new required components approach, which should automatically add the default `Origin`. So let's at least leave a `//INFO:`, or something like that here to remind us to remove this later on.